uxarray/grid/utils.py

```diff
@@ -4,70 +4,6 @@
 import uxarray.utils.computing as ac_utils
 
 
-def _replace_fill_values(grid_var, original_fill, new_fill, new_dtype=None):
-    """Replaces all instances of the current fill value (``original_fill``) in
-    (``grid_var``) with (``new_fill``) and converts to the dtype defined by
-    (``new_dtype``)
-
-    Parameters
-    ----------
-    grid_var : np.ndarray
-        grid variable to be modified
-    original_fill : constant
-        original fill value used in (``grid_var``)
-    new_fill : constant
-        new fill value to be used in (``grid_var``)
-    new_dtype : np.dtype, optional
-        new data type to convert (``grid_var``) to
-
-    Returns
-    ----------
-    grid_var : xarray.Dataset
-        Input Dataset with correct fill value and dtype
-    """
-
-    # locations of fill values
-    if original_fill is not None and np.isnan(original_fill):
-        fill_val_idx = np.isnan(grid_var)
-    else:
-        fill_val_idx = grid_var == original_fill
-
-    # convert to new data type
-    if new_dtype != grid_var.dtype and new_dtype is not None:
-        grid_var = grid_var.astype(new_dtype)
-
-    # ensure fill value can be represented with current integer data type
-    if np.issubdtype(new_dtype, np.integer):
-        int_min = np.iinfo(grid_var.dtype).min
-        int_max = np.iinfo(grid_var.dtype).max
-        # ensure new_fill is in range [int_min, int_max]
-        if new_fill < int_min or new_fill > int_max:
-            raise ValueError(
-                f"New fill value: {new_fill} not representable by"
-                f" integer dtype: {grid_var.dtype}"
-            )
-
-    # ensure non-nan fill value can be represented with current float data type
-    elif np.issubdtype(new_dtype, np.floating) and not np.isnan(new_fill):
-        float_min = np.finfo(grid_var.dtype).min
-        float_max = np.finfo(grid_var.dtype).max
-        # ensure new_fill is in range [float_min, float_max]
-        if new_fill < float_min or new_fill > float_max:
-            raise ValueError(
-                f"New fill value: {new_fill} not representable by"
-                f" float dtype: {grid_var.dtype}"
-            )
-    else:
-        raise ValueError(
-            f"Data type {grid_var.dtype} not supported" f"for grid variables"
-        )
-
-    # replace all zeros with a fill value
-    grid_var[fill_val_idx] = new_fill
-
-    return grid_var
```

Comment:
This shouldn't be moved either. Please refer to the above comment if the circular import issues still persist.

Reply:
I checked, it is not used, what is used is in connectivity